examples/amp-story/video-one-page.html

```diff
@@ -9,6 +9,8 @@
 
   <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
   <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0/amp-viewer-integration-0.1.js"></script>
+  <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
```

Comment:
> We didn't have to manually add these extensions to the test files, I believe this is a regression in the local server

> Error logs when running `amp --compiled` and navigating to the test page

@Enriqe the two things mentioned above are separate from each other.

- Running `amp --compiled` is a local-development-only scenario that lazy-builds extensions as they are fetched by test pages. If a test page fetches an extension or a binary, it will get lazily-built. Otherwise, it will not. (This fact has been true for a long time.)
- When unit / integration / e2e tests are run during CI, we pre-build the AMP runtime and all extensions. So it is guaranteed that binaries like `amp-story-1.0.js`, `amp-video-0.1.js`, `amp-viewer-integration-0.1.js`, and `amp-social-share-0.1.js` are all present in the `dist/` directory. The responsibility of actually fetching them lies with the test pages or the extensions themselves (in case one extension / JS binary auto-fetches another).

**TL;DR:** Lazy-building does not happen during CI, and is unlikely to have anything do to with flakiness. Curious: Were there other symptoms you noticed during CI testing that could point to what changed?